mpp/shmemx_c_func.m4

```diff
@@ -18,3 +18,7 @@ void SHPRE()shmemx_ct_set(shmemx_ct_t ct, long value);
 void SHPRE()shmemx_ct_wait(shmemx_ct_t ct, long wait_for);
 
 void SHPRE()shmemx_pcontrol(int level, ...);
+
+void SHPRE()shmemx_int_bxor(int *target, int value, int pe);
+void SHPRE()shmemx_long_bxor(long *target, long value, int pe);
+void SHPRE()shmemx_longlong_bxor(long long *target, long long value, int pe);
```

Comment:
Almost.  Take a look at ```mpp/shmem_c_func.m4``` for inspiration.  You want to define the function prototype as an m4 macro and then ```SHMEM_DEFINE_FOR_AMO(...)```.

Another comment -- these aren't the right types.  You will also need to add a new type category in ```src/shmem_bind_c.m4``` for the bitwise AMO types.